packages/react-native/Libraries/Blob/URL.js

```diff
@@ -104,51 +100,54 @@ export class URL {
       }
       this._url = `${baseUrl}${url}`;
     }
+
+    // Parsing the URL to use for accessors
+    this._parsedUrl = new globalThis.URL(this._url);
```

Comment:
Why `globalThis` here?

Reply:
Also, can this be done lazily, to avoid parsing if the accessors are not ever used?

Reply:
resolved